Makefile

```diff
@@ -145,7 +145,7 @@ kubectl: ## Configure kubeconfig context for the cluster using `kubectl config`
 kubectl-delete: ## Delete the created CLUSTER_NAME context from the kubeconfig (uses kubectl).
 	$(eval CLUSTERCERTSDIR := $(shell mktemp -d))
```

Comment:
Could you remove this line (and the empty line 147) while you are at it, I just noticed that this isn't needed (not sure why I added it in my change)

Reply:
Done!